### PR TITLE
Add protobuf3-to-dict recipe

### PR DESCRIPTION
**Pull request by setu4993**
Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by setu4993** [comment]
@conda-forge/core : This PR is ready for review.

**Comment by CJ-Wright** [comment]
Please ping @conda-forge/staged-recipes

File: recipes/protobuf3-to-dict/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - protobuf_to_dict
+
+about:
+  home: "https://github.com/benhodgson/protobuf-to-dict"
+  license: Public Domain
+  license_family: PUBLIC-DOMAIN
+  license_file:
+  summary: "A teeny Python library for creating Python dicts from protocol buffers and the reverse. Useful as an intermediate step before serialisation (e.g. to JSON)."
+  doc_url:
+  dev_url:
```
Comment (ocefpaf) [inline recipes/protobuf3-to-dict/meta.yaml]: You can either fill these optional fields os remove them. Don't leave them blank.

**Comment by setu4993** [comment]
Thanks, @ocefpaf, removed those fields now.

**Comment by setu4993** [comment]
@conda-forge/staged-recipes : This PR is ready for review.

**Comment by setu4993** [comment]
@conda-forge/staged-recipes : Bump.

**Comment by setu4993** [comment]
Closing this in lieu of https://github.com/conda-forge/staged-recipes/pull/9970.

**Pull request closed without merging**